x-pack/plugins/apm/public/components/app/mobile/service_overview/index.tsx

```diff
@@ -154,7 +154,7 @@ export function MobileServiceOverview() {
             <EuiFlexGroup gutterSize="s">
               <EuiFlexItem grow={5}>
                 <EuiPanel hasBorder={true}>
-                  <LatencyMap
+                  <MetricsMap
```

Comment:
Do we really need to name it `MetricsMap`? can't it be just ApmMaps?

Reply:
What about `GeoMap` to make it clear it's not about service map.

Reply:
https://github.com/elastic/kibana/pull/150040/commits/477b842c3b8e10f5e51780d0710524edfa657cea